packages/bus-rdf-resolve-hypermedia-links/lib/ActorRdfResolveHypermediaLinks.ts

```diff
@@ -38,6 +38,7 @@ export interface IActorRdfResolveHypermediaLinksOutput extends IActorOutput {
   links: ILink[];
 }
 
+// TODO deprecate this in next major version
```

Comment:
Good!
But let's just re-export the ILink from the types here, to make sure they are always identical.

```suggestion
// TODO remove this in next major version
```